node/core/approval-voting/src/approval_db/v1/mod.rs

```diff
@@ -166,6 +166,8 @@ pub struct OurAssignment {
 	pub validator_index: ValidatorIndex,
 	// Whether the assignment has been triggered already.
 	pub triggered: bool,
+	// Claimed core indices.
```

Comment:
this seems like a dumb comment.

Reply:
True :) 

Doc comments on pall ublic fields would be good